index.ts

```diff
@@ -137,13 +143,24 @@ function cleanupStep(
   const commands = machineDeletes.concat(wait2Mins).concat(volumeDeletes);
   return {
     label: ":broom: Clean up fly resources",
-    commands,
+    key: "cleanup-step",
+    command: commands,
     // TODO(dmiller): instead of hardcoding this, maybe grab the buildkite agent tags from
     // [BUILDKITE_AGENT_META_DATA_*](https://buildkite.com/docs/pipelines/environment-variables#BUILDKITE_AGENT_META_DATA_)
-    agents: "deploy=true",
+    agents: ["deploy=true"],
     depends_on: dependencies,
     allow_dependency_failure: true,
-    plugins: ["thedyrt/skip-checkout#v0.1.1"],
+    plugins: [
+      "thedyrt/skip-checkout#v0.1.1",
+      {
+        "seek-oss/aws-sm#v2.3.1": {
+          region: "us-east-2",
+          env: {
+            FLY_API_TOKEN: "prod/fly-api-token"
+          }
+        },
```

Comment:
would be nice if we could get the region/token secret name from the containing buildkite pipeline step (specifically the step that has this plugin in its list).  is there some way to do that?

Reply:
hm, yeah.  also means more changes everywhere.  actually, I think in the context where we're building the cleanup step, I should have access to `BUILDKITE_PLUGINS` and can ~grovel~ perform some magic there.